### PR TITLE
Adds Playwright unit tests to ProgressBar web component

### PR DESCRIPTION
**Pull request by procload**
## New Behavior

Adds Playwright unit tests to ProgressBar web component

### CODE REVIEW

**Comment by fabricteam** [comment]
## 🕵 fluentui-web-components-v3 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/c83bfec16fd0d93b1305e5f5aec5d8a62c8f7cb7'>c83bfec16fd0d93b1305e5f5aec5d8a62c8f7cb7</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/web-components/src/progress-bar/progress-bar.spec.ts [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    await expect(element).toHaveJSProperty('shape', 'square');
+  });
+
+  test('should set and retrieve the `validationState` property correctly', async () => {
+    await element.evaluate((node: ProgressBar) => {
+      node.validationState = 'warning';
+    });
+
+    await expect(element).toHaveJSProperty('validationState', 'warning');
+  });
```
Comment (chrisdholt) [inline packages/web-components/src/progress-bar/progress-bar.spec.ts:151]: I think we should test for each possible variation of these as they are all part of the public API.

Comment (procload) [reply packages/web-components/src/progress-bar/progress-bar.spec.ts]: Fixed!

**Review by halerankin — APPROVED** [review]

**Review by brianchristopherbrady — APPROVED** [review]

**Review by miroslavstastny — APPROVED** [review]

**Pull request merged**